macos/FluentUI/Button.swift

```diff
@@ -53,24 +66,39 @@ open class Button: NSButton {
 		// Do common initialization work
 		isBordered = false
 		wantsLayer = true
-		
+		layer?.contentsScale = window?.backingScaleFactor ?? 1.0
+
 		self.style = style
-		
+
 		if let title = title {
 			self.title = title
 		}
-		
+
 		if let image = image {
 			self.image = image
 		}
 	}
+
+	open class override var cellClass: AnyClass? {
+		get {
+			return FluentButtonCell.self
+		}
+		set {}
+	}
 	
 	override public var image: NSImage? {
 		willSet {
 			guard wantsLayer == true else {
 				preconditionFailure("wantsLayer must be set so that the image is rendered on the layer")
 			}
 		}
+		didSet {
+			if let image = image {
+				if image.isTemplate {
+					self.image = image.tint(color: self.textColor)
```

Comment:
do we really need this? we can't utilize contentTintColor ?

Reply:
Well, I didn't know about that property :D 
It's only 10.14 up though, so we'll need to have a fallback, for which textColor is fine?

Reply:
So this suggestion was great. I rewrote the text to also get tinted with contentTintColor, with an appropriate 10.13 fallback. Thank you :D 
Images will not be tinted on 10.13